client/ayon_core/tools/loader/abstract.py

```diff
@@ -767,8 +764,6 @@ def get_selected_project_name(self):
             Union[str, None]: Selected project name.
         """
 
-        pass
```

Comment:
Please no, revert all the unnecesasry changes. There is HUGE PR touching exactly the same files and stuff, do as less as possible please.

Reply:
you really want me to return back all the unnecessary `pass`... 🙄 in abstract methods that are not touched anyway. But as you wish.